pkg/files/sources_test.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2022 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package files
```

Comment:
I think package name should be files_test

Reply:
Thanks!